packages/services/api/src/modules/alerts/providers/adapters/msteams.ts

```diff
@@ -72,7 +72,13 @@ export class TeamsCommunicationAdapter implements CommunicationAdapter {
         }),
       );
     } catch (error) {
-      this.logger.error(`Failed to send Microsoft Teams notification`, error);
+      const errorText =
+        error instanceof Error
+          ? error.toString()
+          : typeof error === 'string'
+            ? error
+            : JSON.stringify(error);
```

Comment:
honestly im not 100% sure this conversion logic is necessary.
I couldn't figure out well our logger handles converting unknown types to strings.